language/move-vm/runtime/src/loader.rs

```diff
@@ -255,7 +256,7 @@ impl ModuleCache {
     }
 
     // `make_type` is the entry point to "translate" a `SignatureToken` to a `Type`
-    fn make_type(&self, module: &CompiledModule, tok: &SignatureToken) -> PartialVMResult<Type> {
```

Comment:
Since `BinaryIndexedView` already takes a module with reference (i.e., `BinaryIndexedView<'a>::Module(&'a CompiledModule)`) and has a `Copy` trait, I guess it is OK to pass in a `BinaryIndexedView` instead of a reference.